service/adapter/classes/Configuration.php

```diff
@@ -90,8 +95,9 @@ public function __construct(Logger $logger)
         $this->encryptedApiKey = $this->getEncryptedAPIKey();
         $this->clientKey = $this->getClientKey();
         $this->liveEndpointPrefix = \Configuration::get('ADYEN_LIVE_ENDPOINT_URL_PREFIX');
-        $this->moduleVersion = '3.2.1';
+        $this->moduleVersion = '3.3.0';
         $this->moduleName = 'adyen-prestashop';
+        $this->integratorName = \Configuration::get('ADYEN_INTEGRATOR_NAME', null, null, null, "");
```

Comment:
Are all the parameters needed?

```suggestion
        $this->integratorName = \Configuration::get('ADYEN_INTEGRATOR_NAME');
```

Reply:
I remember checking this, the last param is the default value and that's why all of those are used

Reply:
Yes, the last param is passed as an empty string to ensure that even if the integratorName does not exist, a string value is passed here:

`$this->setExternalPlatform("PrestaShop", _PS_VERSION_, $configuration->integratorName);`

Otherwise, **false** would be passed.

Reply:
got it!